src/java.base/share/classes/sun/net/www/protocol/file/FileURLConnection.java

```diff
@@ -46,6 +47,12 @@ public class FileURLConnection extends URLConnection {
     private static final String TEXT_PLAIN = "text/plain";
     private static final String LAST_MODIFIED = "last-modified";
 
+    // The feature of falling back to FTP for non-local file URLs is disabled
+    // by default and can be re-enabled by setting a system property
+    private static boolean FTP_FALLBACK_ENABLED = "true".equalsIgnoreCase(
+            System.getProperty("jdk.net.file.ftpfallback", "false")
+    );
+
```

Comment:
Maybe we should use `Boolean.getBoolean("jdk.net.file.ftpfallback")` here.

Reply:
I think I did it this way because I wanted the default to be explicit. But, yes Boolean::getBoolean is short and sweet.